cmsis-pack crate does not respect local configuration of root certificates

`Cache` is unable to download packs in enterprise environments with "transparent" TLS-terminating middleboxes:

```
$ pack-manager cache packs
Caching all packs; this will take some time. Press Control-C to quit.
20:57:14 [WARN] Sending fatal alert BadCertificate
20:57:14 [ERROR] https://sadevicepacksprodus.blob.core.windows.net/idxfile/index.pidx: error trying to connect: invalid certificate: UnknownIssuer
```

`reqwest` [supports](https://docs.rs/reqwest/0.10.10/reqwest/#optional-features) bundling Mozilla's root certificates (`rusttls-tls`, i.e., `rust-tls-webpki-roots`) or using the platform's native certificate store (`rustls-tls-native-roots`). See [`rustls-native-certs`](https://github.com/rustls/rustls-native-certs#should-i-use-this-or-webpki-roots) for a comparison of the two approaches.

Consider switching from `rust-tls-webpki-roots` to `rustls-tls-native-roots` to support enterprise environments with "transparent" TLS-terminating middleboxes.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -15,7 +15,7 @@ name = "cmsis_pack_manager"
 crate-type = ["cdylib"]
 
 [dependencies]
-ctor = "0.1.15"
+ctor = "0.2"
 log = "0.4.8"
 simplelog = { version = "0.12.0", default-features = false, features = [ "termcolor" ] }
 cmsis-pack = { version = "0.6.2", path = "rust/cmsis-pack" }

--- a/rust/cmsis-cffi/Cargo.toml
+++ b/rust/cmsis-cffi/Cargo.toml
@@ -15,7 +15,7 @@ name = "cmsis_cffi"
 crate-type = ["cdylib"]
 
 [dependencies]
-ctor = "0.1.15"
+ctor = "0.2"
 log = "0.4.8"
 simplelog = { version = "0.12.0", default-features = false, features = [ "termcolor" ] }
 cmsis-pack = { version = "0.6.2", path = "../cmsis-pack" }

--- a/rust/cmsis-pack/Cargo.toml
+++ b/rust/cmsis-pack/Cargo.toml
@@ -20,7 +20,7 @@ minidom = "0.12.0"
 serde = { version = "1.0.118", features = ["derive"] }
 serde_json = "1.0"
 tokio = { version = "1.0", features = ["macros", "rt"] }
-reqwest = { version = "0.11.0", default_features = false, features = ["rustls-tls", "trust-dns", "stream"] }
+reqwest = { version = "0.11.0", default_features = false, features = ["rustls-tls-native-roots", "trust-dns", "stream"] }
 anyhow = "1.0.56"
 
 [dev-dependencies]

